influxdb3/client.go

```diff
@@ -88,7 +88,10 @@ func New(config ClientConfig) (*Client, error) {
 	c.apiURL.Path = path.Join(c.apiURL.Path, "api/v2") + "/"
 
 	// Prepare auth header value
-	c.authorization = "Token " + c.config.Token
+	if c.config.TokenPrefix == "" {
+		c.config.TokenPrefix = "Token"
+	}
+	c.authorization = fmt.Sprintf("%s %s", c.config.TokenPrefix, c.config.Token)
```

Comment:
The implementation should not modify `config` instance provided by the user.

```suggestion
	tokenPrefix = c.config.TokenPrefix
	if tokenPrefix == "" {
		tokenPrefix = "Token"
	}
	c.authorization = fmt.Sprintf("%s %s", tokenPrefix, c.config.Token)
```